pkg/tms/tms.go

```diff
@@ -286,6 +317,36 @@ func (communicationInstance *CommunicationInstance) UploadFileToNode(nodeName, f
 
 }
 
+func (communicationInstance *CommunicationInstance) ExportFileToNode(nodeName, fileId, description, namedUser string) (NodeUploadResponseEntity, error) {
```

Comment:
ExportFileToNode is almost identical to UploadFileToNode. Maybe it can be exported to a separate function.

Reply:
ok